CHANGELOG.md

```diff
@@ -12,10 +12,15 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
-## v2.0.2 (WIP)
+## v3.0.0 (WIP)
+
+- BREAKING: Removed support for `github.com/iver-wharf/wharf-api` v4.2.0 and
+  below. (#49)
+
+- Added support for `github.com/iver-wharf/wharf-api` v5.0.0. (#49)
```

Comment:
nit: Better to state minimum version. Specifying v4.2.0 feels oddly arbitrary, like question pops up of "is there a v4.3.0 that it does support?"

```suggestion
- BREAKING: Removed support for `github.com/iver-wharf/wharf-api` v4.
  Now requires a minimum of wharf-api v5.0.0. (#49)
```

Same applies for your other PRs of the same changeset: https://github.com/iver-wharf/wharf-provider-github/pull/45 https://github.com/iver-wharf/wharf-provider-gitlab/pull/43